bin/debian-9-base/create-container.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
None of these .sh scripts are set `chmod +x`. Please do so.

Reply:
Good catch. Solved with 7fb7bc9

Reply:
Debian provides the libmozjs185-1.0 package with the same spidermonkey version. Could we just skip the source build and use that?

Reply:
No. We've found some serious segfaults in the libmozjs185-1.0 package and build our own with different flags to avoid these.

Reply:
This bug is actually fixed in the debian package: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=811665

Besides GCC6 prints a lot of possible errors (warnings) in the source code that are partly addressed by the debian packaging team.

I would recommend using the official package - at least for the stretch build as the root of your problems with seem to come from building for arm?

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=839048